test/metabase/analytics/snowplow_test.clj

```diff
@@ -149,10 +149,11 @@
 
       (snowplow/track-event! ::snowplow/database-connection-successful
                              1
-                             {:database :postgres, :database-id 1, :source :admin})
+                             {:database :postgres, :database-id 1, :source :admin, :dbms_verrsion "14.1"})
       (is (= [{:data    {"database" "postgres"
                          "database_id" 1
                          "event" "database_connection_successful"
+                         "dbms_verrsion" "14.1"
```

Comment:
verrsion?

Reply:
IDK, this test feels kind of pointless. Maybe if it tested the endpoint for the database creation for the test being sent - that would make sense. Right now it just tests that mocking underlying mechanism of `track-event!` works, not that DBMS version is tracked.

Reply:
> Maybe if it tested the endpoint for the database creation for the test being sent - that would make sense

yes, I did that in `create-db-succesful-track-snowplow-test` and `create-database-test`

> IDK, this test feels kind of pointless
is it necessary to update this test? no as we can leave the payload as is and it'll still pass, but I think we should keep it as close as the actual schema as possible.

on whether or not it's pointless, I think it's not, it's actually also test if the payload can be serialized, and whether the event-key map to a the correct schema.